### PR TITLE
Add output formats

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**
Add JSON, TABLE, YAML and TEXT output formats

### CODE REVIEW

File: lib/services/output/output-interface.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+interface OutputInterface {
+    print(data: any): void;
```
Comment (marashlyansuren) [inline lib/services/output/output-interface.d.ts:2]: @sergeymirzoyan1991 jan do you think to have another PR for defining the data interface ????

**Review by marashlyansuren — COMMENTED** [review]
@sergeymirzoyan1991 do we need to commit js files ???

File: lib/services/output/output-interface.d.ts (hunk 1 above)
Comment (sergeymirzoyan1991) [reply lib/services/output/output-interface.d.ts:2]: that's the plan, once we start using the engine app here and we know the exact type of this data, will change the type `any` to that specific type

**Review by tsardaryan — APPROVED** [review]
